frontend/appflowy_flutter/lib/plugins/database_view/application/field/type_option/type_option_context.dart

```diff
@@ -143,7 +143,8 @@ class TypeOptionContext<T extends GeneratedMessage> {
   }
 
   set typeOption(T typeOption) {
-    _dataController.typeOptionData = typeOption.writeToBuffer();
+    _dataController.typeOptionData
+     = typeOption.writeToBuffer();
```

Comment:
```suggestion
    _dataController.typeOptionData = typeOption.writeToBuffer();
```

remove whitespace changes